src/main/java/br/com/caelum/vraptor/simplemail/MailerFactory.java

```diff
@@ -35,9 +35,6 @@ public Mailer getInstance() {
 	}
 
 	private Mailer grabInstance() {
-		if (env.getName().equals("development")) {
-			return new MockMailer();
-		}
```

Comment:
@JamesSullivan I totally agree with this change, however releasing it will cause a break of compatibility.
Everyone who updates the version of this plugin will have real emails sent in development environment.
Any thoughts for a soft change? Maybe adding a new key on properties to toggle this. What do u think?